src/Localization/en/translations.json

```diff
@@ -36,6 +36,57 @@
             "SubmissionMessageErrorText": "Data update failed because of errors.",
             "OnSaveFailureMessage": "Data save operation not currently permitted."
         },
-        "SqlX": {}
+        "SqlX": {
+            "DedicatedGatewayDescription": "Provision a dedicated gateway cluster for your Azure Cosmos DB account. A dedicated gateway is compute that is a front-end to data in your Azure Cosmos DB account. Your dedicated gateway automatically includes the integrated cache, which can improve read performance. ",
+            "DedicatedGateway": "Dedicated Gateway",
+            "Enable": "Enable",
+            "Disable": "Disable",
+            "LearnAboutDedicatedGateway": "Learn more about dedicated gateway.",
+            "DeprovisioningDetailsText": "Learn more about deprovisioning the dedicated gateway.",
+            "DedicatedGatewayPricing": "Learn more about dedicated gateway pricing",
+            "SKUs": "SKUs",
+            "SKUsPlaceHolder": "Select SKUs",
+            "NumberOfInstances": "Number of instances",
+            "CosmosD4s": "Cosmos.D4s (General Purpose Cosmos Compute with 4 vCPUs, 16 GB Memory)",
```

Comment:
>Cosmos.D4 [](start = 26, length = 9)

if i do a search in search engine for cosmos.D4, i dont  see anything.. May be "Cosmos D4 (General....)"  instead of enum style Cosmos.D4?